pkg/common/constants.go

```diff
@@ -32,8 +32,10 @@ const (
 	MonitoringNamespace = "everest-monitoring"
 	// PerconaEverestDeploymentName stores the name of everest API Server deployment.
 	PerconaEverestDeploymentName = "percona-everest"
+	// PerconaEverestCatalogName is the name of the Everest catalog source.
+	PerconaEverestCatalogName = "everest-catalog"
 	// PerconaEverestOperatorDeploymentName stores the name of everest operator deployment.
-	PerconaEverestOperatorDeploymentName = "everest-operator-controller-manager"
+	PerconaEverestOperatorDeploymentName = "everest-operator"
```

Comment:
I'm so glad that the helm chart brought this along :pray: I hated the name :joy: